packages/stable/src/types.ts

```diff
@@ -1656,6 +1656,124 @@ export interface ProjectUpdate {
   authentication?: InputProjectAuthentication;
 }
 
+/**
+* Change that will be applied to the array.
+*/
+export interface ArrayPatchString {
+    set?: Set<string>;
+    remove?: Set<string>;
+    add?: Set<string>;
+}
+
+export interface OidcConfigurationUpdate {
+    modify?: OidcConfigurationUpdateDTO;
+    set?: OidcConfigurationDTO;
+    setNull?: boolean;
+}
+
+export interface ClaimName {
+    claimName?: string;
+}
+
+export interface ModifyPatchInteger {
+    set?: number;
+    setNull?: boolean;
+}
+
+export interface OidcConfigurationUpdateDTO {
+    jwksUrl?: SetField<string>;
+    tokenUrl?: SinglePatchString;
+    issuer?: SetField<string>;
+    audience?: SetField<string>;
+    skewMs?: ModifyPatchInteger;
+    accessClaims?: ModifyPatchClaimNames;
+    scopeClaims?: ModifyPatchClaimNames;
+    logClaims?: ModifyPatchClaimNames;
+}
+
+export interface ModifyPatchClaimNames {
+    set?: Array<ClaimName>;
+    add?: Array<ClaimName>;
+    remove?: Array<ClaimName>;
+}
+
+export interface ProjectUpdateDTO {
+    update: ProjectUpdateObjectDTO;
+}
+
+/**
+* Contains the instructions on how to update the project. Note: azureADConfiguration, oidcConfiguration and oAuth2Configuration are mutually exclusive
+*/
+export interface ProjectUpdateObjectDTO {
```

Comment:
```suggestion
export interface ProjectUpdateObject {
```
We don't use `DTO` postfix